src/base.config.ts

```diff
@@ -239,11 +282,12 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 						{
 							loader: 'css-loader',
 							options: {
-								modules: true,
-								sourceMap: true,
+								getLocalIdent,
 								importLoaders: 1,
 								localIdentName: '[name]__[local]__[hash:base64:5]',
-								getLocalIdent
+								modules: true,
+								sourceMap: true,
+								url: args.target !== 'lib'
```

Comment:
what does this do?

Reply:
It prevents `css-loader` from replacing `url()` values for library builds.

Reply:
don't we want to resolve url's for library builds? why should the css be different in this mode?

Reply:
Since I imagine we want libraries to be able to use the `url(~from/node_modules)` syntax, then we do need to resolve the URLs. At the time, I experienced some difficulty getting a clean match between the replaced `url()` path and the actual location of the asset. For example, if the CSS file is `theme/icon.m.css`, then the font it imports would be output to `output/{mode}/name.woff`, but `output/{mode}/theme/icon.m.css` would import it as `url(name.woff)` instead of `../name.woff`. Fortunately, a newer version of `MiniCssExtractPlugin` allows the `publicPath` option to be a function, which effectively mitigates this issue (so the font would be imported as `url(../name.woff)`).